app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -159,17 +159,24 @@
           %td.cya-question
             = t('jobs.contact_email')
           %td.cya-answer
-            =mail_to 'Job contact email', @vacancy.contact_email, 'aria-label': t('jobs.aria_labels.contact_email_link', email: @vacancy.contact_email)
-          %td.cya-change
-            = link_to t('buttons.change'), application_details_school_job_path(anchor: 'contact_email'), 'aria-label': t('jobs.aria_labels.change_job_contact_email')
+            - if @vacancy.contact_email.present?
+              =mail_to 'Job contact email', @vacancy.contact_email, 'aria-label': t('jobs.aria_labels.contact_email_link', email: @vacancy.contact_email)
+              %td.cya-change
+                = link_to t('buttons.change'), application_details_school_job_path(anchor: 'contact_email'), 'aria-label': t('jobs.aria_labels.change_job_contact_email')
```

Comment:
@despo I think we've introduced a bug here that hides the change link in either state.